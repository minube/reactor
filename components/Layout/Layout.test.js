import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import LayoutView from './Layout';

describe('<LayoutView>', () => {
  it('renders', () => {
    const tree = renderer.create(<LayoutView />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('when {onLayout}', () => {
    // @TODO: Test <LayoutView /> {onLayout}
  });

  it('renders in DOM environment', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LayoutView />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
});

