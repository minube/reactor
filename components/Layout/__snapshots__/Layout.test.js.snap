// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LayoutView> renders 1`] = `
<View
  onLayout={undefined}
  style={
    Array [
      Object {
        "height": "100%",
        "overflow": "hidden",
        "width": "100%",
      },
      undefined,
    ]
  }
/>
`;

exports[`<LayoutView> when {style} 1`] = `
<View
  onLayout={undefined}
  style={
    Array [
      Object {
        "height": "100%",
        "overflow": "hidden",
        "width": "100%",
      },
      Object {
        "backgroundColor": "rgba(0,255,0,0.25)",
      },
    ]
  }
/>
`;
